test/graphql/Offer/queries/getOfferByUuid.test.ts

```diff
@@ -209,5 +227,20 @@ describe("getOfferByUuid", () => {
       });
       expect(errors![0].extensions!.data).toEqual({ errorType: AuthenticationError.name });
     });
+
+    it("throws if user is not admin, approved company or approved applicant", async () => {
```

Comment:
Aca escribiria esto como 
`returns an error if the user is not admin, approved company or approved applicant`